packages/flutter/lib/src/widgets/icon.dart

```diff
@@ -35,7 +35,8 @@ class Icon extends StatelessWidget {
   const Icon(this.icon, {
     Key key,
     this.size,
-    this.color
+    this.color,
+    this.label,
```

Comment:
I actually think label is not the best name here, as it doesn't imply accessibility, @goderbauer what do you think?
maybe accessibilityLabel? or contentDescription?

Reply:
You could call it semanticsLabel. We are avoiding the term accessibility in the API, as this semantic description of the UI could also be used for other purposes.

Reply:
You could also add to the doc comment of label that this label is not shown in the UI.